lib/mix/lib/mix/cli.ex

```diff
@@ -114,7 +114,8 @@ defmodule Mix.CLI do
   end
 
   defp display_version() do
-    IO.puts "Mix #{System.version}"
+    #IO.puts :erlang.system_info(:system_version)
```

Comment:
oh.. yeah. thanks for reminding me that one.

```sh
$ elixir --version
Erlang/OTP 18 [erts-7.0] [source] [smp:2:2] [async-threads:10] [hipe] [kernel-poll:false]

Elixir 1.3.0-dev (b587871)
$ iex --version
Erlang/OTP 18 [erts-7.0] [source] [smp:2:2] [async-threads:10] [hipe] [kernel-poll:false]

IEx 1.3.0-dev
$ mix --version
Mix 1.3.0-dev
```

Should we make it consistent with the other commands?